pulp_2to3_migration/app/pre_migration.py

```diff
@@ -3,7 +3,10 @@
 from collections import namedtuple
 
 from django.db import transaction
-from django.db.models import Max
+from django.db.models import (
```

Comment:
Nitpick: why split across 2 lines?

Reply:
Just for the sake of keeping the style (if we have any). If there is more than one import, we split it across multiple lines. I can put it in one line, not a problem.

Reply:
Ah, does black require that now?

Reply:
Not sure, have not checked that